app/views/users/new.html.erb

```diff
@@ -0,0 +1,92 @@
+<!--
+# <form action="/users" method="post" accept-charset="UTF-8">
+#   <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+#   <label for="username">Username:</label>
+#   <input type="text" id="username" name="user[username]"><br /><br />
+#   <label for="email">Email:</label>
+#   <input type="text" id="email" name="user[email]"><br /><br />
+#   <label for="password">Password:</label>
+#   <input type="password" id="password" name="user[password]"><br /><br />
+#  <input type="submit" value="Submit">
+# </form>
+-->
+
+<!--
+#<%= form_tag("/users", method: "post") do %>
```

Comment:
- [ ] Instead of hard coding the user path in here use the `user_path` helper instead. Getting used to path helpers will come in handy when you work with more complex and scoped paths that are super easy to make typos in.
  - Use the user_path also in the regular HTML form.